modin/core/dataframe/pandas/partitioning/partition_manager.py

```diff
@@ -60,9 +60,18 @@ def wait(*args, **kwargs):
                 partitions = result[0]
             else:
                 partitions = result
+            # When partitions have a deferred call queue, calling
```

Comment:
I think this comment should be rephrased taking into account these discussions - [link1](https://github.com/modin-project/modin/pull/4472#issuecomment-1136042874) and [link2](https://github.com/modin-project/modin/pull/4472#issuecomment-1136219274).

Reply:
What would you change it to? It seems like as is, it accurately explains why the `drain_call_queue` is there.

Reply:
I read again this and probably would change the last part a bit. Left a suggestion.